src/plugins/intel_gna/src/gna2_model_export_helper.cpp

```diff
@@ -278,3 +215,40 @@ void ExportTlvModel(uint32_t modelId,
         THROW_GNA_EXCEPTION << "Not succesfull status returned: " << tlv_status << ", from Gna2ExportTlv() function\n";
     }
 }
+
+void* ExportSueLegacyUsingGnaApi2(uint32_t modelId, uint32_t deviceIndex, Gna2ModelSueCreekHeader* modelHeader) {
```

Comment:
Correct mi if I am wrong, purpose of this method is to copy header of model to modelHeader variable and at the same time return dump.

Reply:
This is out of scope of this PR.